scalikejdbc-interpolation/src/test/scala/scalikejdbc/QueryInterfaceSpec.scala

```diff
@@ -353,7 +353,7 @@ class QueryInterfaceSpec extends FlatSpec with Matchers with DBSettings with SQL
         {
           val notInClauseResults = withSQL {
             select.from(Order as o)
-              .where.not.in(o.id, Seq())
+              .where.not.in(o.id, Seq[Int]())
```

Comment:
Maybe we can use same trick in `TxBoundaryMissingImplicits` ?

https://github.com/scalikejdbc/scalikejdbc/blob/2.3.5/scalikejdbc-core/src/main/scala/scalikejdbc/TxBoundary.scala#L46-L47